src/tools/pkcs15-tool.c

```diff
@@ -57,6 +57,7 @@ typedef unsigned __int32 uint32_t;
 #include "libopensc/pkcs15.h"
 #include "libopensc/asn1.h"
 #include "util.h"
```

Comment:
clang-format suggested the following change:


```
+#include "util.h"
```

<!-- identifier: clang-format -->
<!-- File: src/tools/pkcs15-tool.c -->